src/app/home/search/search.component.ts

```diff
@@ -14,9 +18,10 @@ enum SearchTypeValue {
   selector: "app-home-search",
   templateUrl: "./search.component.html",
   styleUrls: ["./search.component.less"],
-  encapsulation: ViewEncapsulation.None
+  encapsulation: ViewEncapsulation.None,
```

Comment:
are you sure about ViewEncapsulation ? this is a really rare case when None is a good option to use. Basically you're applying component styles to all app